packages/main/src/plugin/webview/webview-panel-impl.ts

```diff
@@ -146,11 +148,16 @@ export class WebviewPanelImpl implements WebviewPanel {
     }
   }
 
-  reveal(preserveFocus?: boolean): void {
+  reveal(_preserveFocus?: boolean): void {
     this.assertNotDisposed();
 
     // notify the renderer to reveal the webview
-    this.#apiSender.send('webview-panel-reveal', { id: this.#internalId, preserveFocus });
+    this.#apiSender.send('navigate', {
+      page: NavigationPage.WEBVIEW,
+      parameters: {
+        id: this.#internalId,
+      },
+    } as NavigationRequest);
```

Comment:
not sure why we're adding `as NavigationRequest` ?

Reply:
Refactored as suggested, agree it's better than `as NavigationRequest`.